EpsilonCCS/Inc/CanParsers/BmsCanParser.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "stm32f4xx.h"
+
+void parseBmuCanMessage(uint32_t stdId, uint8_t data[8]);
```

Comment:
Yeah, the only thing I really prefer about it though is it gets rid of the magic number 8.